tests/components/AdviceGeneratorApp.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {render, screen} from '@testing-library/react';
+import { server } from '../../src/mocks/server';
+import { rest } from 'msw';
+import { AdviceGeneratorApp } from '../../src/components/AdviceGeneratorApp/AdviceGeneratorApp';
+
+
+describe('tests on <AdviceGeneratorApp />', () => {
+
+  test('should show the initial state', () => {
+
+    render(<AdviceGeneratorApp/>);
+    expect(screen.getByText('Loading Advice...'))
+
```

Comment:
Puedes revisar si el advice se ha pintado en la UI:

```suggestion
    expect(screen.getByText('Loading Advice...'))
    
    const adviceEl = await screen.findByText("Smile and the world smiles with you. Frown and you're on your own.");
    expect(adviceEl).toBeTruthy()
```